migrate_pim/upgrade_major_version.rst

```diff
@@ -0,0 +1,36 @@
+How to upgrade to a major version?
+==================================
+
+We release a major version each year, these new major versions bring new features and larger changes to answer clients growing needs.
+
+Migrate from 1.7 to 2.0
+------------------------------
+
+To migrate from 1.7 to 2.0, we recommend the use of our brand new migration tool `Transporteo`_.
+
+.. _Transporteo: https://github.com/akeneo/transporteo
+
+We're continuously improving Transporteo to cover more and more use cases and automate more and more the migrations.
+
+Migrate from 2.3 to 3.0
+------------------------------
+
+To migrate from 2.0 to 3.0 version, you first need to go through minor versions.
```

Comment:
I'd say: "To migrate from 2.0 to 3.0 version, It is necessary to do the progressively upgrade process to each 2.X version."